src/reference/asciidoc/whats-new.adoc

```diff
@@ -114,3 +114,9 @@ For this purpose the `ErrorMessageExceptionTypeRouter` loads mapping classes dur
 for a `ClassNotFoundException`.
 
 See <<router>> for more information.
+
+==== Header Mapping
+
+AMQP, WS and XMPP header mappings (e.g. `request-header-mapping`, `reply-header-mapping`) now support negated
```

Comment:
From other side we don't announce that all our header mappers support negate :smile: 
So, we can reconsider support for all other in the next iteration or even version.